pagefaultgames/pokerogue
# [Localization][Bug] Korean Font update and expand coverage

## What are the changes?
Large Korean texts (using `emerald`) become slightly smaller, like Alphabets.
Texts are more fit to UI.

Small Korean texts (using 'pkmnes') are no longer large-unreferenced-clipped glyphs.

## Why am I doing these changes?
It can be 2nd version of #1678.
This font file is generated for pokerogue, so it is also an unfinished product and has issues.

I want to make overall look better.

## What did change?
Update `PokePT_Wansung.ttf` to new version.
No change, only the glyph size shrinks (8/9).

In `i18n.ts`, one more *font-face* is loaded when locale is Korean.
Cuz of *unicodeRange*, ASCII range is not changed.
Only large texts are changed to pixel style font.


### Screenshots/Videos
Orange marks are drawn to help understanding.

* Smaller text
![image](https://github.com/pagefaultgames/pokerogue/assets/87186129/dd3170b5-8f64-4935-aaac-a63e6f76f72b)
Left one is old, center is new. Right one is English version for comparison.
Text heights are same.

* Pixelated text
![image](https://github.com/pagefaultgames/pokerogue/assets/87186129/0da6505d-e702-40f5-a14a-8e06c86f2fd3)
`pkmnems` font-family is now loaded. Left one is old.
You can also see ♀ and other word have same height.

* Clear pokemon names
![image](https://github.com/pagefaultgames/pokerogue/assets/87186129/dfe6b58b-fd07-4f6a-a406-c4d2a3c3a59f)
From left to right, before loading font-face(old one), font-face loaded, and font file updated(got smaller).
In summary ui. Names are its right position, not just none-clipped.

* Modifier select UI
![image](https://github.com/pagefaultgames/pokerogue/assets/87186129/d6501371-30dd-4956-b282-d3143d5807f0)
![image](https://github.com/pagefaultgames/pokerogue/assets/87186129/e445ddcf-3819-4802-abc0-7be82da24388)
Upper one is old, Lower is new.
Item name is not overlapped by its price.

## How to test the changes?
`emerald` font is everywhere. ASAP turn on the game, result is shown.
`pkmnems` is more rare. Differences are in:
* `same` Daily run : "Score"
* center`diff`, corner`same` Reward after battle : All text except in the chatbox
* name`diff`, other`same` Pokemon Summary : Name, level, ♂♀, HP counter, cancel button in the corner.
* `diff` Pokemon Catching and starter start UI : Pokemon move list items.
* + `diff` starter select UI : Right side texts in the pokemon stat box.
* `same` Pokemon level in Save&Load UI
* `diff` Gacha UI : Result rarity text.

## Checklist
